include/psp2kern/usbserial.h

```diff
@@ -33,26 +33,26 @@ int sceUsbSerialStartForDriver(void);
  *
  * @param unk Unknown
  */
-int sceUsbSerialSetupForDriver(int unk);
+int sceUsbSerialSetupForKernel(int unk);
 
 /**
  * Get USB device status
  *
  * @return Returns 1 when serial port is open
  */
-int sceUsbSerialStatusForDriver(void);
+int sceUsbSerialStatusForKernel(void);
 
 /**
  * Close USB serial device
  */
-int sceUsbSerialCloseForDriver(void);
+int sceUsbSerialCloseForKernel(void);
 
 /**
  * Get receive buffer length
  *
  * @return Returns length of USB rx buffer
  */
-unsigned int sceUsbSerialGetLenForDriver(void);
+unsigned int sceUsbSerialGetLenForKernel(void);
```

Comment:
Oh yeah we wanted to change that. 
sceUsbSerialGetRXLenForKernel is what comes to mind. To me